components/dropdown/style/index.tsx

```diff
@@ -456,9 +456,9 @@ export default genComponentStyleHook(
       dropdownEdgeChildPadding: paddingXXS,
     });
     return [
+      genStatusStyle(dropdownToken),
```

Comment:
> I prefer to remain the order because `genBaseStyle` should be at first. Maybe problem is solved without changing order since the priority has been raised?

I've also tested this and it still doesn't fix the problem without reordering

Reply:
I made some changes without reordering the functions